tests/runtime/test_http_server.py

```diff
@@ -350,7 +350,7 @@ def test_health_check_ok(runtime_http_server):
         assert response.text == "OK"
 
 
-def test_pydantic_wrapping_with_enums():
+def test_pydantic_wrapping_with_enums(runtime_http_server):
```

Comment:
I'm curious why this changed since `_dataobject_to_pydantic` is a `@classmethod` and doesn't need an instance of the server. This seems like it's adding extra work by booting a real server, although I believe `runtime_http_server` is session scoped, so it's probably not _actually_ extra work.

Reply:
Yeah I guess that makes sense. I think it was added only for convenience so that we don't have to create an instance, but unnecessary overhead for sure